size.go

```diff
@@ -37,22 +37,34 @@ var (
 var decimapAbbrs = []string{"B", "kB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"}
 var binaryAbbrs = []string{"B", "KiB", "MiB", "GiB", "TiB", "PiB", "EiB", "ZiB", "YiB"}
 
-// CustomSize returns a human-readable approximation of a size
-// using custom format.
-func CustomSize(format string, size float64, base float64, _map []string) string {
+func customSize(size float64, base float64, _map []string) (float64, string) {
```

Comment:
Thanks @dnephin. The PR has been updated to `getSizeAndUnit`. Let me know if there are any issues.